ci/docs/rdg-parser.py

```
@@ -0,0 +1,391 @@
+    def _parse_yaml_config(self, content: str, configurations: Dict):
+        """Parse YAML configuration blocks"""
+        try:
+            data = yaml.safe_load(content)
+            if data and isinstance(data, dict):
+                # Check for service configurations
+                if 'kind' in data:
+                    kind = data['kind']
+                    if kind not in configurations['service_configurations']:
+                        configurations['service_configurations'][kind] = []
+                    configurations['service_configurations'][kind].append(data)
+        except:
```

Flake8 (7.2.0)

[error] do not use bare 'except'

(E722)